pages/index.js

```diff
@@ -111,14 +110,7 @@ const HomePage = () => (
                   <p>
                     Run on <code>http://localhost:3000/</code>:
                   </p>
-                  <pre>
-                    yarn sketchbook start {"<path/to/design/file.json>"}
-                  </pre>
-                  <p>
-                    Replace <code>{"<path/to/design/file.json>"}</code> with{" "}
-                    <code>sketchbook/example.json</code> to run it with an
-                    example design file.
-                  </p>
+                  <pre>yarn sketchbook start example.json</pre>
```

Comment:
This is incorrect still, it generates the files at `sketchbook/example.json` (as stated in the previous step):

```suggestion
                  <pre>yarn sketchbook start sketchbook/example.json</pre>
```